### PR TITLE
Make IdentitySerde handle bytes as well

### PR DESCRIPTION
**Pull request by srkukarni**
### Motivation

This will allow bytes to go thru the system unchanged

### Modifications

Describe the modifications you've done.

### Result

After your change, what will change.

### CODE REVIEW

File: pulsar-client-cpp/python/pulsar/functions/serde.py [hunk 1]
```diff
@@ -75,6 +75,8 @@ def __init__(self):
   def serialize(self, input):
     if type(input) in self._types:
       return str(input).encode('utf-8')
+    if type(input) == bytes:
+      return input
     raise TypeError
```
Comment (merlimat) [inline pulsar-client-cpp/python/pulsar/functions/serde.py]: We should also add context on the type that we got passed here

Comment (srkukarni) [reply pulsar-client-cpp/python/pulsar/functions/serde.py]: Added

**Comment by srkukarni** [comment]
rerun integration tests

**Comment by srkukarni** [comment]
rerun java8 tests

**Pull request merged**